drones/connection/prototype.py

```diff
@@ -0,0 +1,22 @@
+"""Basic drone connection and communication.
+
+Additional info.
+"""
+
+import logging
+
+
+class ConnectorPrototype:
+    def __init__(self):
+        self.log = logging.getLogger(__name__)  # Get global logger setup

```

Comment:
Loggers should be defined outside of class methods unless you have a good reason for it. Also, I recommend formatting it at least with the module name (same way you did it in `main.py`).